tests/testprogs/uprobe_fork_loop.c

```diff
@@ -0,0 +1,36 @@
+#include <unistd.h>
+
+int GLOBAL_A = 0x55555555;
+int GLOBAL_B = 0x88888888;
+int GLOBAL_C = 0x33333333;
+char GLOBAL_D = 8;
+
+int uprobeFunction1(int *n, char c __attribute__((unused)))
+{
+  return *n;
+}
+
+void spin()
+{
+  while (1)
+  {
+    int n = 13;
+    char c = 'x';
+    uprobeFunction1(&n, c);
+    usleep(500);
+  }
+}
+
+int main(int argc __attribute__((unused)), char **argv __attribute__((unused)))
+{
+  if (fork() == 0)
+  {
+    spin();
+  }
+  else
+  {
+    spin();
+  }
```

Comment:
`spin()` is run in both the true and false cases - can the if-statement be removed entirely?

Reply:
Good call. It can. This is what I get from just copying from the interwebs